### PR TITLE
[LENS-1324] Adds examples on getting trackingId and deviceId

### PR DESCRIPTION
**Pull request by npushkarskii**
This PR adds examples on how to:

* set a new `deviceId`

* get the current `deviceId` 

* get the current `trackingId`

https://coveord.atlassian.net/browse/LENS-1324

### CODE REVIEW

**Review by jsonDaCruzo — APPROVED** [review]
Thanks @npushkarskii, please wait for @hporter approval as he is a better judge on the code samples provided.

**Review by hporter — APPROVED** [review]

**Pull request merged**
